src/test/components/__snapshots__/SampleTooltipContents.test.js.snap

```diff
@@ -261,18 +273,6 @@ exports[`SampleTooltipContents renders the sample tooltip properly 1`] = `
         class="backtraceStackFrameOrigin"
       />
     </li>
-    <li
-      class="backtraceStackFrame backtraceStackFrame_isFrameLabel"
-    >
-      <span
-        class="colored-border category-color-grey"
-        title="Other"
-      />
-      nsAppStartup::Run
-      <em
-        class="backtraceStackFrameOrigin"
-      />
-    </li>
```

Comment:
Oh, I was confused how this could be removed from the stack, it turns out this was because of the max stacks limit that is 20 since we added another frame on top of it.

Reply:
right, I didn't even notice, thanks for checking!